client/state/users/reducer.js

```diff
@@ -1,34 +1,9 @@
-/**
- * External dependencies
- */
-
 /**
  * Internal dependencies
  */
-
-import suggestions from './suggestions/reducer';
 import { combineReducers } from 'calypso/state/utils';
-import { CURRENT_USER_RECEIVE } from 'calypso/state/action-types';
-
-/**
- * Tracks user objects, indexed by user ID.
- *
- * @param  {object} state  Current state
- * @param  {object} action Action payload
- * @returns {object}        Updated state
- */
-export function items( state = {}, action ) {
-	switch ( action.type ) {
-		case CURRENT_USER_RECEIVE:
-			return Object.assign( {}, state, {
-				[ action.user.ID ]: action.user,
-			} );
-	}
-
-	return state;
-}
+import suggestions from './suggestions/reducer';
```

Comment:
Should we instead pop this out to a root-level `userSuggestions` piece of state? It seems unrelated to the rest of user/currentUser state.

It may be best left for a future PR, though.

Reply:
I was planning to add a sibling `state.users.items` reducer here but yeah, moving `state.users.suggestions` to a standalone root reducer is a good idea. Let's do that in a followup. There will be several followups here anyway.

Reply:
I'll take care of the followup on this one, @jsnajdr 👍

Reply:
Thank you! I expect that the consumers of `state.users.suggestions` and `state.users.items` will be two disjunct sets with little overlap, so it makes sense to modularize them as independent reducers?

Reply:
> Thank you! I expect that the consumers of `state.users.suggestions` and `state.users.items` will be two disjunct sets with little overlap, so it makes sense to modularize them as independent reducers?

Yes, definitely! With the modularised Redux model we have in place, finer granularity is the way to go 👍